HLTrigger/Muon/plugins/HLTScoutingMuonProducer.cc

```diff
@@ -0,0 +1,179 @@
+// -*- C++ -*-
```

Comment:
HLTrigger/Muon already defines only plugins (in the /src directory), you should not use a new /plugins directory, just put the .cc files in the /src directory.